### PR TITLE
Add review invitation endpoints

### PR DESCRIPTION
**Pull request by fabubaker**
Adds the following endpoints and their docs:

* `POST /invite`
* `POST /revoke`
* `POST /restore`
* `POST /delete_review_access`

### CODE REVIEW

File: .insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  | 2       | Collaborator |
+
+  | 3       | Viewer       |
+
+  | 4       | Translator   |
+
+  | 6       | Reviewer     |
+
+
+  * `user_emails`: A list of emails of the invitees.
```
Comment (hammady) [inline .insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml:29]: Clarify the format (e.g CSV)

**Review by hammady — CHANGES_REQUESTED** [review]
Document the feature flag necessary to perform any action.

File: .insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  | 2       | Collaborator |
+
+  | 3       | Viewer       |
+
+  | 4       | Translator   |
+
+  | 6       | Reviewer     |
+
+
+  * `user_emails`: An array of strings containing the emails of the invitees. Eg: `["user1@example.com", "user2@example.com"]`.
```
Comment (hammady) [inline .insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml:29]: Is this the JSON equivalent for URL-encoded content type like this one:
image

Comment (fabubaker) [reply .insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml:29]: Wdym? The content-type is set to JSON, so your typical JSON arrays will be understood by Rails. 

image

Comment (hammady) [reply .insomnia/Request/req_591ce65b6fd948d68770be96b6c0346a.yml:29]: OK, it is clear now. I was only comparing it to the web endpoint and the `/customizations` but just realized that one is a GET request and has no JSON body.

**Review by hammady — APPROVED** [review]

**Pull request merged**